jetson/db/data_insertion.py

```diff
@@ -0,0 +1,24 @@
+from ftp_transfer import ftp_transfer
+from db_connection import sql_insert, IMAGE, BBOX
+
+
+def data_insert(image_name, image_date, image_time, init_vec, bboxes, input_dir):

```

Comment:
can you add type hinting to this function. The docstring outlines it but having the type inline helps with readability.

example:

`def data_insert(image_name: str, image_date: datetime, ... )`